index.md

```diff
@@ -1,11 +1,244 @@
+3. No later than the month following DP2, the RSP hybrid model should be technically reviewed to determine whether it is sound in its present form and if not, any related risks should be exercised so that mitigations can be in place in time for DR1. This item also reinforces (1) and (2).
+4. In the context of the scope of this document, the Prompt Products release is entirely un-rehearsed and requires significant attention from Data Management to get right. Given the obvious high interest in the community for timely transient science, this release should be given equivalent priority (if not higher) than DP2. This item also reinforces (2).
+5. Agency or other sign-off should be sought to make even a ComCam-sized subset of the data "ok public" (if not actually public) to bypass restrictions that prevent us from shipping data for CI purposes to your cloud-hosted services.
+
+For recommendations at the Data Management group levels and the reasoning for the key recommendations, see individual sections.
+
+## Problem summary
+
+The overall symptom of the problems identified was:
+
+* The interval between data production of DP1 and live release on the RSP took 4 months of elapsed effort, instead of the anticipated 3 months.
+* In the three month plan, there was 1 month of contingency which was fully exhausted.
+
+The release was on time regardless due to heroic efforts, a minor fortuitous delay in First Light release, slipping some issues unlikely to be immediately impacting users till after going live, deferring non-DP1-related tasks, do-overs, and fixing bugs in the wild (mostly found by staff testing) that could have been caught ahead of time had contingency not eaten into the testing time.
+
+A number of these interventions would have been ineffective for a DR1-scale event.
+
+The eventual goal should be a 3-month worst-case release consisting of 2 months of release work _including_ contingency, and 1 month of testing/bugfixing prior to release. We are unlikely to reach this goal in the next two releases.
+
+## High-level issues, and related recommendations
+
+
+### Handover checkpoints
+
+While it's not quite as simple, there are six major transfers that can serve as checkpoints after data production is complete:
+
+1. Pipelines to Qserv (Catalogs)
+2. Pipelines to Middleware (Images)
+3. Pipelines to [Data Services? Middleware?] (HiPS)
+4. Qserv to/from Data Engineering ()
+5. Data Engineering to SQuaRE/Portal
+6. Data Services to CST
+
+In the data previews so far this has been done "bucket-brigade" style with teams leads at each transfer point co-ordinating the bucket-passing between themselves, as opposed to the Czar model of a single person managing all the steps.
```

Comment:
I think we should be taking a critical view of the "bucket brigade" itself.  In my view there was inadequate "pipelining" of the entire flow, leaving us with implied "take it or leave it" handoffs from one team to the next, when some investment in overlapped work would have produced earlier upstream feedback and required less re-work.